conda_envs/acinoset.yml

```diff
@@ -22,3 +22,4 @@ dependencies:
     - pyqt5
     - tables
     - nptyping==0.3.1
+    - tqdm
```

Comment:
tqdm is already on line 8. Do you need it to be installed specifically by pip?

Reply:
Yes. Tqdm did not work with previous acinoset.yml.